src/styles/main.scss

```
@@ -1 +1,82 @@
-// put code here
+$clock-center-size: 40px;
+$arrow-second-size: $clock-center-size / 3;
+$arrow-minut-size: $clock-center-size / 2;
+
+body {
+  margin: 0;
+}
+
+.clock {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+  justify-content: center;
+
+  height: 100vh;
+
+  &__content {
+    position: relative;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    height: 80vh;
+    width: 80vh;
+    box-sizing: border-box;
+
+    border: 8px dotted #000;
```

create a variable for colors
```suggestion
    border: 8px dotted $c-black;
```